src/util.zig

```diff
@@ -19,6 +19,11 @@ pub fn sliceOfInstanceConst(comptime T: type, instance: *const T) []const T {
     return slice;
 }
 
+pub fn asConstSentinelTerminated(string_ptr: [*:0]u8) [:0]const u8 {
```

Comment:
This is good but I think we should also have asStringSlice. Maybe follow the new zig format, fromCstringToStringSlice?

Reply:
Done, I've added a CStringToStringSlice and I'm using that (instead of the 0-terminated one)